docs/users/docker_installation.md

```diff
@@ -0,0 +1,62 @@
+<h1>Docker Installation</h1>
+
+## macOS Installation: Docker-ce For Mac
+
+Most MacOS versions and computers will run Docker for Mac. Homebrew users can `brew cask install docker` or you can download from [download.docker.com](https://download.docker.com/mac/stable/Docker.dmg). 
```

Comment:
Given that we tend to bias/prefer Docker for Mac, should we promote that first in the 2nd sentence? Maybe "Docker For Mac is the preferred docker environment for macOS. Alternatively, you can use the Homebrew option by running `brew cask install docker`."

Reply:
Well, we *only* support Docker-for-mac, so that's the only product mentioned here. Let me know what you think is ambiguous and I can improve it. (Docker-ce == Docker-for-mac == docker-ce-for-mac).  They still have docker toolbox for Mac but we've never tried it and haven't ahd any reason to. It might work. Maybe a couple of those really, really old macs we ran into at Badcamp. Not sure.

Reply:
Sure. I guess what tripped me us I almost always went for the DMG approach, but if brew cask is installing the CE edition than my comments are irrelevant. Sorry for the noise there!

Reply:
Thanks, I do wish the command was `brew cask install docker-ce`, not sure why it isn't. If this is OK then please approve and we'll get it in.